packages/flutter_tools/test/general.shard/vmservice_test.dart

```diff
@@ -3,362 +3,164 @@
 // found in the LICENSE file.
 
```

Comment:
Since we don't setup the peer ourselves anymore, testing this is a bit simpler - though these tests as written aren't very meaningful right now